### ISSUE TITLE
rainbow saber(?) causes beat saber to soft lock

### ISSUE THREAD

**Issue by Spootin23** [RedBrumbler/Qosmetics#35]
Not sure exactly what the cause was, or if the saber choice had anything to do with it. I turned on the option that allows you to use your saber model in the menu. it was working fine, until I switched sabers, now I have no pointer laser. I cant select anything, and the pointer laser only shows up for a split second, when switching pointer hand, but not for long enough to allow a selection to be made. i have tried turning the mod off and on, and even uninstalling and reinstalling. but whenever qosmetics is turned on, i start with the rainbow sabers, and cant select anything. I love this mod, and it is one of the most interesting ways to change the game, please help me out if you can so i can use it again. Thanks for all the hard work, i appreciate it.

**Comment by RedBrumbler** [comment]
Delete those sabers from the Qosmetics/sabers folder in the moddata folder on your quest

**Comment by Spootin23** [comment]
Of course. I was trying to look for the folder they were in but I was having a little trouble finding it. Would you mind telling me the file path?
________________________________

**Comment by RedBrumbler** [comment]
Should be sdcard/ModData/com.beatgames.beatsaber/Mods/Qosmetics/sabers

**Comment by Spootin23** [comment]
Muchos Gracias, I'll get to that tonight, if you don't hear from me then there were no issues. Thanks for the help.
________________________________
